metrics/tarantool/network.lua

```diff
@@ -11,19 +11,13 @@ local function update_network_metrics()
 
     collectors_list.net_sent_total =
         utils.set_counter('net_sent_total', 'Totally sent in bytes', box_stat_net.SENT.total)
-    collectors_list.net_sent_rps =
```

Comment:
@yngvar-antonsson , I have thought that we already removed all rps metrics.

Reply:
We deprecated it in 0.13.0